README.md

```diff
@@ -84,7 +84,7 @@ Yup, that's it. You can proxy requests to the Go service through Apache/nginx/et
 
 ## Support
 
-Currently Hound is only tested on MacOS and CentOS, but it should work on any *nix system. Hound on Windows is not supported but we've heard it compiles and runs just fine.
+Currently Hound is only tested on MacOS and CentOS, but it should work on any *nix system. Hound on Windows is not supported but we've heard it compiles and runs just fine, just remember to exclude your data folder from Windows Search Indexer and possibly the virus scanner.
```

Comment:
Nit: I'm a little hesitant about telling people to have a virus scanner exclude the data folder, especially since it could be full of 3rd party code. I also added some minor phrasing tweaks, but I like this otherwise.
```suggestion
Currently Hound is only tested on MacOS and CentOS, but it should work on any *nix system. Hound on Windows is not supported but we've heard it compiles and runs just fine (although it helps to to exclude your data folder from Windows Search Indexer).
```

Reply:
I thought everyone excluded their IDE + Build tools, and source code folders.    The default windows antivirus/malware tool (MsMpEng.exe) was definitely battling hard to realtime scan the 1200+ repos that  20 searcher/indexers were downloading/processing.  The problem should be obvious enough to anyone else using this on windows if they look at the task manager so I have no complaints about the edit..